gen2-play-encoded-stream/main.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+import depthai as dai
+import subprocess as sp
+from os import name as osName
+
+# Create pipeline
+pipeline = dai.Pipeline()
+
+# Define sources and output
+camRgb = pipeline.createColorCamera()
+videoEnc = pipeline.createVideoEncoder()
+xout = pipeline.createXLinkOut()
+
+xout.setStreamName("h265")
+
+# Properties
+camRgb.setBoardSocket(dai.CameraBoardSocket.RGB)
+camRgb.setResolution(dai.ColorCameraProperties.SensorResolution.THE_4_K)
+videoEnc.setDefaultProfilePreset(3840, 2160, 30, dai.VideoEncoderProperties.Profile.H265_MAIN)
+
+# Linking
+camRgb.video.link(videoEnc.input)
+videoEnc.bitstream.link(xout.input)
+
+width, height = 720, 500
+cmd_out = ["ffplay", "-i", "-", "-x", str(width), "-y", str(height)]
+if osName == "nt":  # Running on Windows
+    cmd_out = ["cmd", "/c"] + cmd_out
+
+# Connect to device and start pipeline
+with dai.Device(pipeline) as device:
+    # Output queue will be used to get the encoded data from the output defined above
+    q = device.getOutputQueue(name="h265", maxSize=30, blocking=True)
```

Comment:
Set to 8 frames and non blocking
This will allow for slow ffplay consumption to drop frames instead of block and buildup

Reply:
By setting `maxSize=8` and `blocking=False` I get a weird bug:

![image](https://user-images.githubusercontent.com/59799831/132344392-b26a1a37-08c2-4462-a67e-9880611d4f39.png)

Console output:
`[hevc @ 0000019747f6f3c0] Stream #0: not enough frames to estimate rate; consider increasing probesize`
`[hevc @ 000001974e9b7a00] Could not find ref with POC 22`

Reply:
@MOj0 frames are dropping in this case - you can increase keyframe frequency to reduce artifacts when frames drop (VideoEncoder), and/or set to h264 encoding as is usually easier to decode (more HW support)

Might also make sense to increase maxSize to a higher value - 8 might be too little, depending on how ffplay consumes data.